### PR TITLE
fix(coredump): ignore fwupd due to issue in Ubuntu

### PR DESCRIPTION
**Pull request by bentsi**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Review by juliayakovlev — DISMISSED** [review]

**Review by KnifeyMoloko — DISMISSED** [review]
LGTM

File: sdcm/coredump.py [hunk 1]
```diff
@@ -301,11 +301,12 @@ def get_list_of_cores(self) -> Optional[List[CoreDumpInfo]]:
         #     Tue 2020-01-14 16:18:56 +07    6321  1000  1000   3 present   /usr/bin/scylla
         #     Tue 2020-01-14 16:31:39 +07   18554  1000  1000   3 present   /usr/bin/scylla
         for line in result.splitlines():
-            if "bash" in line:
+            if "bash" in line or "fwupd" in line:
```
Comment (fgelcer) [inline sdcm/coredump.py:304]: `"bash" in line` not sure is a good idea, as there might be in a real coredump as well

**Review by fgelcer — DISMISSED** [review]
LGTM

**Comment by rayakurl** [comment]
IMO need to add #TODO comment to remove this change after the Ubuntu issue is fixed

**Comment by fruch** [comment]
> IMO need to add #TODO comment to remove this change after the Ubuntu issue is fixed

or maybe the other way around, i.e. filter only one that has scylla in the executable name, we don't care much about other parts of the system.

**Comment by bentsi** [comment]
> aybe the other way around, i.e. filter only one that has scylla in the executable name, we don't care much about other parts of the system.
@fruch 
It's a bit tricky since we have other Scylla dependant services that can produce core dump (scylla-jmx for example, and many others)

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**